### DISCUSSION TITLE
[Help]: Intersection observer doesn't work in firefox < 69 version

### DISCUSSION THREAD

**Discussion by Enkratia** [davidjerleke/embla-carousel#766] [Help] [answered]
### Summary

Is there any examples how to make embla work in firefox < 69 version? 
I don't quite understand how to combine polyfill with emla.

### If applicable, which variants of Embla Carousel are relevant to this question?

- [ ] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
@Enkratia thanks for your question. Firefox `v68` and all the way down to `v55` should [support](https://caniuse.com/intersectionobserver) `IntersectionObserver`:

![IMG_7824](https://github.com/davidjerleke/embla-carousel/assets/11529148/fa35bc03-9aa8-43c9-8e62-8c6dcabb8ed5)

**Reply by davidjerleke** [reply, marked as answer]
@Enkratia did you mean `ResizeObserver` maybe? Because that’s not supported in Firefox < 69:

![IMG_7825](https://github.com/davidjerleke/embla-carousel/assets/11529148/b8863f31-a036-426f-84dc-55df008ffde2)

If yes, you can set the option `watchResize: false` and write your own resize handler.

Best,
David

**Reply by Enkratia** [reply]
Thank you
